homeassistant/components/homeassistant_yellow/config_flow.py

```diff
@@ -92,7 +92,7 @@ async def async_step_install_zigbee_firmware(
             firmware_name="Zigbee",
             expected_installed_firmware_type=ApplicationType.EZSP,
             step_id="install_zigbee_firmware",
-            next_step_id="confirm_zigbee",
+            next_step_id="pre_confirm_zigbee",
```

Comment:
This didn't match the other device flows that uses the hardware flow.

Reply:
Good catch. The `pre_confirm_zigbee` step exists solely to stop `user_input` from a previous step from being passed to `confirm_zigbee`, skipping the confirmation. I never was able to figure out how or why this happens.